src/main/resources/com/adobe/jenkins/github_pr_comment_build/TriggerPRCommentBranchProperty/config.jelly

```diff
@@ -5,4 +5,7 @@
     <f:entry title="Comment Body" field="commentBody">
         <f:textbox />
     </f:entry>
-</j:jelly>
\ No newline at end of file
+    <f:entry title="Allow Untrusted users to trigger the build" field="allowUntrusted">
+        <f:checkbox default="false"/>
```

Comment:
I think we should maintain backwards compatibility and make the default true. This can't be done just with this checkbox, though, as that will not set it for everyone that already has the plugin configured. It's possible we need to flip the property to be `preventUntrusted` or something so that the default is false (previous version behavior) and checking it will start enforcing it for those that want it.

Reply:
I would disagree with this but don't want to discuss it in public due to obvious reaons